### ISSUE TITLE
Is this being maintained? It simply doesn't work.

### ISSUE THREAD

**Issue by flowstate** [DataDog/fluent-plugin-datadog#6]
**Describe what happened:**
Followed precisely the steps in the readme. No matter how I get the message to fluentd, it won't ship anything to datadog, and even with -vv enabled, there is no output other than a loop of this:

`2018-05-16 08:48:18 -0500 [trace]: #0 fluent/log.rb:281:trace: enqueueing all chunks in buffer instance=70184845832440`
`2018-05-16 08:48:23 -0500 [trace]: #0 fluent/log.rb:281:trace: enqueueing all chunks in buffer instance=70184845832440`
`2018-05-16 08:48:29 -0500 [trace]: #0 fluent/log.rb:281:trace: enqueueing all chunks in buffer instance=70184845832440`
`2018-05-16 08:48:30 -0500 [trace]: #0 fluent/log.rb:281:trace: Send nb_event=1 events to Datadog`
`2018-05-16 08:48:30 -0500 [trace]: #0 fluent/log.rb:281:trace: Datadog plugin: about to send event=fp`

**Describe what you expected:**
For it to work.

**Steps to reproduce the issue:**
Follow the documentation.

**Additional environment details (Operating System, Cloud provider, etc):**
MacOS high sierra.

**Comment by NBParis** [comment]
Hello @flowstate ,

Sorry for the delay here. 
Sad to hear that it does not work for you. 

Would it be possible to give us more information about your FluentD configuration so we can investigate your issue?

Also could you confirm that the port 10516 is not blocked on your instance?

Thanks,
Nils

**Comment by nathan-oakes** [comment]
+1, but only on one host (using same config for lots of nodes). Super weird.

**Comment by NBParis** [comment]
Hello ,

We do understand that there might be issues on a specific server but to help us investigate we would need to have more details about the configuration for that server.

@nathan-oakes Would it be possible to send your configuration to support@datadoghq.com so we can investigate more? (as you said, it works fine on other nodes so we probably have to investigate more why it doesn't on this specific host).

We are going to close this issue as we got the confirmation that this plugin works for a lot of users and we are still waiting for more details about the original configuration that had an issue.

That said, do not hesitate to open a new issue if you ever faced a problem with that plugin. The more details you add, the more it helps us to investigate.

Thanks again for helping us improving Datadog.
